isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml

```diff
@@ -53,8 +54,8 @@
        cameras (ONC-W1,ONC-W2). 
```

Comment:
The telescopic camera is also supported

Reply:
Reference to ONC-T is made on the line above this.